code/modules/mob/living/carbon/human/species_types/oozelings.dm

```diff
@@ -49,8 +49,12 @@
 		H.blood_volume += 5
 		H.adjustBruteLoss(5)
 		to_chat(H, "<span class='danger'>You feel empty!</span>")
+	if(H.nutrition >= NUTRITION_LEVEL_ALMOST_FULL)
+		H.adjust_nutrition(-5)
+		if(H.blood_volume <= 672)
+			H.blood_volume += 10
 	if(H.nutrition >= NUTRITION_LEVEL_WELL_FED && H.blood_volume <= 672)
-		H.blood_volume += 1
+		H.blood_volume += 8
```

Comment:
Being almost full will run both of these (no else if), for a total of 18 blood per tick. Not sure if that's intentional or not

Reply:
This was intentional, if it needs adjusting however I'll tweak it a bit

Reply:
Well, it's pretty damn fast, especially for a race that can utilize high blood levels

Reply:
adjusting it to
```
	if(H.nutrition >= NUTRITION_LEVEL_WELL_FED && H.blood_volume <= 672)
		if(H.nutrition >= NUTRITION_LEVEL_ALMOST_FULL)
			H.adjust_nutrition(-5)
			H.blood_volume += 10
		else
			H.blood_volume += 8
			```